### DISCUSSION TITLE
怎么使用Json格式的CQ码?

### DISCUSSION THREAD

**Discussion by ghost** [Mrs4s/go-cqhttp#1404] [Q&A] [answered]

**Comment by ghost** [comment]
已解决...

**Reply by qianjunakasumi** [reply]
您可将您的解决方法及其相关资料附至说明中，感谢您对社区的贡献

**Comment by ghost** [comment, marked as answer]
{
    "group_id":888888888,
    "message":[{
        "type": "face",
        "data": {
            "id": "123",
        }
    },{
        "type": "text",
        "data": {
            "text":"发送内容~~~~"
        }
    }]
}
